docs/upgrading.asciidoc

```diff
@@ -0,0 +1,49 @@
+[[upgrading]]
+=== Upgrading APM Server
```

Comment:
Have you thought about putting the upgrade content at the top-level in the TOC (to be consistent with other docs in the stack)? It's fine to do it this way as long as you think users will see the content.